<launch>
    <arg name="use_bag_file" default="false"/>
    <arg name="bag_folder_path" default="/path/to/bagfile"/>
    <arg name="bag_file_name" default="your_bagfile.bag"/>

    <arg name="use_label_file" default="false"/>
    <arg name="gt_label_folder_path" default="/path/to/ground_truth/label_folder"/>
    <arg name="pred_label_folder_path" default="/path/to/pred/label_folder"/>

    <arg name="start_frame" default="-1"/>
    <arg name="end_frame" default="-1"/>

    <node pkg="awv_mos" type="mos_evaluation"      name="mos_evaluation"       output="screen">
        <param name="use_bag_file" value="$(arg use_bag_file)"/>
        <param name="bag_file_path" value="$(arg bag_folder_path)$(arg bag_file_name)"/>
        <param name="use_label_file" value="$(arg use_label_file)"/>
        <param name="gt_label_folder_path" value="$(arg gt_label_folder_path)"/>
        <param name="pred_label_folder_path" value="$(arg pred_label_folder_path)"/>
        <param name="start_frame" value="$(arg start_frame)"/>
        <param name="end_frame" value="$(arg end_frame)"/>
        <param name="scan_topic" value="/awv_mos/segmented_query_scan_all"/>
        <param name="result_folder_path" value="$(find awv_mos)/history/evaluation_results/"/>
    </node>
</launch>
